protos/vehicle_data.proto

```diff
@@ -186,9 +186,12 @@ enum Field {
   Experimental_13 = 176;
   Experimental_14 = 177;
   Experimental_15 = 178;
+
+  // fields below here are always returned typed
+  DetailedChargeState = 179;
```

Comment:
@agbpatro Hey, we are seeing this value returned in the logs in-place of this field. `DetailedChargeState`. How do we actually access this value and should this line be included?  https://github.com/teslamotors/fleet-telemetry/issues/226

Reply:
`l.Debugw("Received value", message.Data[i].Key.String(), message.Data[i].Value.GetStringValue())`
Received value: `"179":""`

Reply:
For other values we get `"ChargeLimitSoc":"100"`